_tests/integration/trigger_params_test.go

```diff
@@ -22,31 +22,45 @@ func Test_TriggerParams(t *testing.T) {
 		require.NoError(t, err, out)
 	}
 
-	t.Log("deprecated trigger with push-branch")
+	t.Log("trigger with push-branch")
 	{
 		cmd := command.New(binPath(), "trigger", "--push-branch", "code-push", "--config", "trigger_params_test_bitrise.yml")
 		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
 		require.NoError(t, err, out)
 	}
 
-	t.Log("deprecated trigger with pr-source-branch")
+	t.Log("trigger with pr-source-branch")
 	{
 		cmd := command.New(binPath(), "trigger", "--pr-source-branch", "pull_request_source_branch", "--config", "trigger_params_test_bitrise.yml")
 		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
 		require.NoError(t, err, out)
 	}
 
-	t.Log("deprecated trigger with pr-target-branch")
+	t.Log("trigger with pr-target-branch")
 	{
 		cmd := command.New(binPath(), "trigger", "--pr-target-branch", "pull_request_taget_branch", "--config", "trigger_params_test_bitrise.yml")
 		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
 		require.NoError(t, err, out)
 	}
 
-	t.Log("deprecated trigger with tag")
+	t.Log("trigger with tag")
 	{
 		cmd := command.New(binPath(), "trigger", "--tag", "tag", "--config", "trigger_params_test_bitrise.yml")
 		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
 		require.NoError(t, err, out)
 	}
+
+	t.Log("deprecated pipeline trigger with pattern")
+	{
+		cmd := command.New(binPath(), "trigger", "--pattern", "deprecated_pipeline_trigger", "--config", "trigger_params_test_bitrise.yml")
+		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+		require.Error(t, err, out)
+	}
+
+	t.Log("pipeline trigger with push-branch")
+	{
+		cmd := command.New(binPath(), "trigger", "--push-branch", "pipeline_trigger", "--config", "trigger_params_test_bitrise.yml")
+		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+		require.Error(t, err, out)
+	}
```

Comment:
Remove these, CLI will never feature pipeline triggering, no need to define what it can't do.